lib/cache/Cache.js

```diff
@@ -44,14 +45,33 @@ class Cache {
         return loc
     }
 
+    async write(filepath, data) {
+        const loc = this.getCacheLocation(filepath)
+        await mkdirp(path.dirname(loc))
+        try {
+            fs.writeFileSync(loc, data)
+        } catch (e) {
+            reporter.debug('Cache.write failed', e)
+        }
+    }
+
+    async read(path) {
```

Comment:
Can we call this variable something other than `path` so we don't override the `import path from 'path'` in this scope?

Reply:
Yes, will change to `filepath` or `fp` or something else. I always forget that `path` is a module so thanks for catching this.

Reply:
I'm strongly against variable names like `fp`, there's no benefit in using abbreviations in most cases and might cause confusion when reading code as one has to think about what it might be first.

Reply:
I used `filepath` for the `write` function so will use the same for `read`.